client/src/App.js

```diff
@@ -1,17 +1,24 @@
 import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
+import { Redirect } from "react-router";
 
 import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+//import LandingPage from "./pages/Landing";
```

Comment:
Try to avoid commented out code in PRs - you can always use github to go to older commits, so you don't need commented out code

Reply:
After testing the calling sign up API from the front end, it should be time to tidy up the codes first :)

Reply:
Done